nservicebus/sagas/concurrency.md

```diff
@@ -151,6 +151,12 @@ In the above example, the requests are split into two groups each time. Dependin
 
 A simpler approach is to split the request just once and have a single level of sub-sagas sending the requests and aggregating the responses. This will not reduce data contention to the same degree, since the originating saga will be aggregating more multiple responses.
 
+#### Sequential scatter/gather
+
+When using a [scatter-gather](https://www.enterpriseintegrationpatterns.com/patterns/messaging/BroadcastAggregate.html) pattern or similar, simultaneously sending a large number of requests may result in simultaneous processing of a large number of responses, which may lead to data contention. Instead, consider sending requests sequentially. In each iteration, send the next request only after the response from the previous request was processed. This approach removes data contention when processing responses but may increase both the overall duration and the size of saga state required to store data while waiting for responses.
+
+Usually, the tradeoff is that the message that initiated the scatter/gather contains data that must now be stored in the saga state so that it can be added to each outgoing request in each iteration.
```

Comment:
@adamralph Added this based on your comment would this help? Not sure if this has value.

Reply:
@ramonsmits yes, that helps, but it needs to live alongside the statement that size increases in the previous paragraph. I'll make a suggestion.